.github/workflows/release.yml

```diff
@@ -5,8 +5,15 @@ on:
     branches:
       - master
 
+permissions:
+  contents: read
+
 jobs:
   release:
+    permissions:
+      # for changesets/action
+      contents: write
+      pull-requests: write
```

Comment:
@sashashura this workflow has only one job, couldn't we use only one `permissions` definition?